NEWS.md

```diff
@@ -59,6 +59,7 @@
     - `atoxgr_criteria_ctcv4_uscv` (NCI-CTCAEv4 criteria)
     - `atoxgr_criteria_ctcv5_uscv` (NCI-CTCAEv5 criteria)
     - `atoxgr_criteria_daids_uscv` (DAIDs criteria)
+- New function `my_first_fcn` to print a welcome message to the console
```

Comment:
```suggestion
- New function `my_first_fcn` to print a welcome message to the console (#1839)
```

We like to put the issue number at the end of the news entry so then it becomes link from news to github issue